Packets/AGENT/INVENTORY/0x3038 - AGENT_INVENTORY_ENTITY_EQUIP.cs

```diff
@@ -0,0 +1,4 @@
+4    uint    GID            // GID of the player who equips the item
+1    byte    slot           // slot the item gets equipped to
+4    uint    uniqueId       // unique id of the item that gets equipped
+1    bool    isOnehanded    // is item that gets equipped a onehanded weapon
```

Comment:
This is actually `1   byte    OptLevel`

Reply:
You mean instead of the last line?
Could you also tell me what the `OptLevel` is? Looking at [this](https://github.com/DummkopfOfHachtenduden/SilkroadDoc/wiki/S_ObjectEquipSet#parameters) packet, I guess it's the "Plus" value of an item?

Reply:
yes and yes